fe_utils/finite_elements.py

```diff
@@ -19,7 +19,11 @@ def lagrange_points(cell, degree):
 
     """
 
-    raise NotImplementedError
+    if cell.dim == 1: # 1D case: equispaced points in [0, 1]
+        return np.array([[i/degree] for i in range(degree+1)])
+        
+    else: # 2D case: {(i/p, j/p) | i <= i+j <= p}
```

Comment:
Technically you should probably raise an exception if someone passes you a cell with a different dimension (e.g. what happens if I pass you a cube?)